src/components/conversations/ParticipantsView.tsx

```diff
@@ -25,23 +50,58 @@ const ParticipantsView: React.FC<ParticipantsViewProps> = (
     );
   }
 
+  const theme = useTheme();
+  // Display only a limited number of participant avatars
+  // Hide the rest behind a text element with a tooltip
+  const maxDisplayedParticipants =
+    props.maxDisplayedParticipants ?? DEFAULT_MAX_DISPLAYED_PARTICIPANTS;
+
+  const displayedParticipants = [];
+  const hiddenParticipants = [];
+
+  for (let i = 0; i < props.participants.length; i++) {
+    const participant = props.participants[i];
+    const name = fetchName(participant);
+    if (i < maxDisplayedParticipants) {
+      displayedParticipants.push(name);
+    } else {
+      hiddenParticipants.push(name);
+    }
+
+    if (i == MAX_HIDDEN_PARTICIPANTS - 1) {
+      // Limit hidden participants too, as we don't want the tooltip to grow up to 1000+ participants large
+      hiddenParticipants.push("...");
+      break;
+    }
+  }
+
   return (
     <>
-      <Box
-        style={{
-          height: "100%",
-          display: "flex",
-          alignItems: "center",
-        }}
-        color="colorTextWeak"
-        fontFamily="fontFamilyText"
-        fontSize="fontSize30"
-        lineHeight="lineHeight40"
-        fontWeight="fontWeightNormal"
-        paddingRight="space60"
+      <Stack
+        orientation={["vertical", "horizontal", "horizontal"]}
+        spacing="space30"
       >
-        {`${props.participants.length} participants`}
-      </Box>
+        <Button variant={"reset"} onClick={props.onParticipantListOpen}>
+          <AvatarGroup names={displayedParticipants} />
+        </Button>
+        {hiddenParticipants.length > 0 ? (
+          <Tooltip
+            text={hiddenParticipants.join(", ")}
+            placement={"bottom-start"}
+          >
+            <span
+              style={{
+                verticalAlign: "top",
+                paddingRight: 10,
+                color: theme.textColors.colorText,
+                fontWeight: theme.fontWeights.fontWeightSemibold,
+              }}
+            >
+              and {hiddenParticipants.length} other Participants
```

Comment:
probably smth like
```
and {props.participants.length - displayedParticipants.length - hiddenParticipants.length}  other Participants
```
also display it only if the value > 0?

Reply:
fixed ✅